extensions/ccui/layouts/UILayout.js

```diff
@@ -384,6 +384,7 @@ ccui.Layout = ccui.Widget.extend(/** @lends ccui.Layout# */{
             rect[3] = cc.p(0, size.height);
             var green = cc.color.GREEN;
             this._clippingStencil.clear();
+            this._clippingStencil.setLocalBB && this._clippingStencil.setLocalBB(cc.rect(0, 0, size.width, size.height));
```

Comment:
If it's possible, please make `setLocalBB` to use 4 parameters instead of an object, to avoid object creation.